expressEngine/db/db.js

```diff
@@ -1,14 +1,15 @@
-import { connect, disconnect } from "mongoose";
+import { connect, disconnect } from 'mongoose';
 
 const dbUrls = {
-  local   : "mongodb://localhost:27017/rnd",
+  local: 'mongodb://localhost:27017/rnd',
 };
+console.log(process.env.MONGO_URL);
```

Comment:
Remove unnecessary console statement.

Reply:
You are reviewing my previous commit, kindly review my latest commit